pkg/election/election.go

```diff
@@ -31,9 +31,13 @@ import (
 )
 
 var (
+	ErrMustReelect     = errors.New("proposals stale, must reelect")
```

Comment:
nit:
```suggestion
	errMustReelect     = errors.New("proposals stale, must reelect")
```
this does not seems to used outside of the package